daos-stack/daos
# DAOS-5101 test: Added dmg pool evict test and updated pool_list_cont

This test covers the following requirement.
(SRS-12-0114) All connections to a specific pool can be revoked via the dmg tool

Test step:
1. Create 2 pools.
2. Create a container in each pool.
3. Call dmg pool evict on the second pool.
4. Call daos pool list-cont on the evicted pool. It should fail with -1012.
5. Verify the other pool isn't affected by creating a new container, list, and destroy.

Also updated pool_list_cont to process output and return dictionary. Updated some tests
that use this method.

Test-tag: pr,-hw dmg_pool_evict list_containers
Test-tag-hw-medium: pr,hw,medium,ib2 enospc_lazy_fg enospc_time_fg enospc_no_aggregation
Signed-off-by: Makito Kano <makito.kano@intel.com>
